benchmarks/fastrnns/test_bench.py

```diff
@@ -13,7 +13,7 @@ def pytest_generate_tests(metafunc):
     # This creates lists of tests to generate, can be customized
     if metafunc.cls.__name__ == "TestBenchNetwork":
         metafunc.parametrize('net_name', all_nets, scope="class")
-        metafunc.parametrize("executor_and_fuser", ["legacy-old"], scope="class")
+        metafunc.parametrize("executor_and_fuser", ["legacy-old", "profiling-te"], scope="class")
```

Comment:
Even though I added this in the first place, I feel uncomfortable with it because I was seeing some odd behavior when running a graph through both the legacy and profiling executor in the same script.  As far as I could tell, switching the executor didn't actually take effect (which is bizarre, and inexplicable, so *maybe* I was doing something wrong).

All this to say, how would you feel about taking the `conftest.py` approach from pytorch/benchmark and using a custom command line option for the fuser/executor, rather than adding it as a fixture?  Alternatively, convince me that I was temporarily insane when I thought I was seeing the wrong executor being used!